core/backend/src/IModelDb.ts

```diff
@@ -1230,15 +1231,13 @@ export abstract class IModelDb extends IModel {
   }
 
   /** Get the IModel coordinate corresponding to each GeoCoordinate point in the input */
-  public async getIModelCoordinatesFromGeoCoordinates(props: string): Promise<IModelCoordinatesResponseProps> {
-    const resultString = this.nativeDb.getIModelCoordinatesFromGeoCoordinates(props);
-    return JSON.parse(resultString) as IModelCoordinatesResponseProps;
+  public async getIModelCoordinatesFromGeoCoordinates(props: IModelCoordinatesRequestProps): Promise<IModelCoordinatesResponseProps> {

```

Comment:
Document breaking public API change here and below in NextVersion.md.

Reply:
added to NextVersion.